src/main/webapp/js/instructorStudentEdit.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready(function(){
+    $("#button_submit").on('click', changeStudnetInformationConfirmation);
```

Comment:
I wonder if you can add a check to show the confirmation dialogue only if the team is edited? Changes to other student information shouldn't cause responses to be deleted.